### PR TITLE
Collect OneMineRate metrics in Kafka module

### LINKED ISSUE #17295: In Metricbeat Kafka modules based on Jolokia, collect OneMinuteRate instead of MeanRate
**Describe the enhancement:**
- For Metricbeat Kafka Jolokia module, we should be able to use different mbean attributes (other than the default MeanRate) 
- I think OneMinuteRate makes much more sense than MeanRate as a default value
- Maybe make it configurable?

**Describe a specific use case for the enhancement or feature:**
By default, it seems like Metricbeat Kafka Jolokia module takes MeanRate attribute of metrics. A lot of times, I would prefer using OneMinuteRate to using the MeanRate. For example, `name=BytesOutPerSec,type=BrokerTopicMetrics`. MeanRate would have mean value of the throughput from broker start time. I want Metricbeat to ingest more granular value, and do necessary aggregation using lastic.

### PR DESCRIPTION
**Pull request by gquintana**
- Bug

## What does this PR do?

Collect OneMinuteRate metrics in Kafka broker metricset instead of MeanRate

## Why is it important?

MeanRate is rate an average rate since Kafka startup.
In short it doesn't mean anything and doesn't shown Kafka current state.

## Checklist

- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Closes #17295 

## Use cases

Kafka monitoring

### CODE REVIEW

**Comment by andresrc** [comment]
@jsoriano can you take a look? would this be a breaking change?

**Comment by jsoriano** [comment]
> @jsoriano can you take a look? would this be a breaking change?

Sure, taking a look. Generally speaking it would be a breaking change, but I agree that current behaviour is not very useful, so it can be considered more a bugfix. Also this metricset is in beta state, so no problem with this change.

@gquintana thanks a lot of this PR! Could you please add a changelog entry in `CHANGELOG.next.asciidoc`?

**Comment by jsoriano** [comment]
jenkins run the tests please

File: metricbeat/module/kafka/broker/manifest.yml [hunk 1]
```diff
@@ -11,81 +11,81 @@ input:
         attributes:
           - attr: Value
             field: request.channel.queue.size
-      - mbean: 'kafka.server:name=FailedProduceRequestsPerSec,type=BrokerTopicMetrics'
+      - mbean: 'kafka.server:type=BrokerTopicMetrics,name=FailedProduceRequestsPerSec'
```
Comment (jsoriano) [inline metricbeat/module/kafka/broker/manifest.yml:14]: @gquintana was this change needed?

**Review by jsoriano — COMMENTED** [review]
Change LGTM, waiting for the changelog and CI and it would be ready to merge. Thanks!

**Comment by jsoriano** [comment]
jenkins run the tests please

File: metricbeat/module/kafka/broker/manifest.yml (hunk 1 above)
Comment (gquintana) [reply metricbeat/module/kafka/broker/manifest.yml:14]: No it's not needed, but
* it makes more readable, as names get vertically aligned (you'll notice there are metric duplication)
* it matches Kafka documentation https://kafka.apache.org/documentation/#monitoring

**Comment by gquintana** [comment]
thanks @jsoriano for the changelog, you were faster than me

File: metricbeat/module/kafka/broker/manifest.yml (hunk 1 above)
Comment (jsoriano) [reply metricbeat/module/kafka/broker/manifest.yml:14]: Sounds good, thanks :+1:

**Review by jsoriano — APPROVED** [review]
